CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 # 1.1.1 (2017-10-31)
 
 - Bugfix when looking for modules globally. [#138](https://github.com/blackbaud/skyux-cli/pull/138)
+- Only logging some information when the `--verbose` flag is used. [#140](https://github.com/blackbaud/skyux-cli/pull/140)
```

Comment:
"Only logging some information" sounds weird to me. I'd suggest changing this entry to "Limited the information that the `--verbose` flag logs."

Reply:
I can see that, but that statement sounds backwards. Including the verbose flag expands the information that’s logged.

Reply:
My bad. I thought "Only logging some" meant that we were limiting the amount of information being. (i.e., switching to only some of the info instead of all the info.)

Would it be accurate to say "Updated the `--verbose` flag to log additional information."?

Reply:
How about "Updated to use the `--verbose` flag to log additional information" or "Using the `--verbose` flag to log additional information."  I suggest those as I didn't update the flag, but updated to use the flag to control what information is logged.

Reply:
Makes sense to me. I like "Use the `--verbose` flag to log additional information."